Source/ZoomNet/Resources/IPhone.cs

```diff
@@ -0,0 +1,51 @@
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+using ZoomNet.Models;
+
+namespace ZoomNet.Resources
+{
+	/// <summary>
+	/// Allows you to access Zoom Phone API endpoints.
+	/// </summary>
+	/// <remarks>
+	/// See <a href="https://developers.zoom.us/docs/api/rest/reference/phone/methods/#overview">
+	/// Zoom API documentation</a> for more information.
+	/// </remarks>
+	public interface IPhone
+	{
+		#region Recordings endpoints
+
+		/// <summary>
+		/// Get recording of a specific phone call by its call ID or call log ID.
+		/// </summary>
+		/// <param name="callId">The call ID or call log ID.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The requested <see cref="PhoneCallRecording" />, if any.
+		/// </returns>
+		/// <remarks>
+		/// See <a href="https://developers.zoom.us/docs/api/rest/reference/phone/methods/#operation/getPhoneRecordingsByCallIdOrCallLogId">
+		/// Zoom endpoint documentation</a> for more information.
+		/// </remarks>
+		Task<PhoneCallRecording> GetRecordingAsync(
+			string callId, CancellationToken cancellationToken = default);
+
+		/// <summary>
+		/// Get transcript of a specific phone call recording by its recording ID.
+		/// </summary>
+		/// <param name="recordingId">The call recording ID.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The requested <see cref="JsonObject"/>, if any.
+		/// </returns>
+		/// <remarks>
+		/// See <a href="https://developers.zoom.us/docs/api/rest/reference/phone/methods/#operation/phoneDownloadRecordingTranscript">
+		/// Zoom endpoint documentation</a> for more information.
+		/// </remarks>
+		Task<JsonObject> GetRecordingTranscriptAsync(
```

Comment:
Is there a reason you chose `JsonObject` rather than create a new model class (something like `RecordingTranscript`)?

Reply:
You know, I haven't really thought about making a specific class for it!
Mind, I might have been biased, as we had plans on just dumping the result in a file (I should be able to simply dump it back to json on my end, of course).

I could add specific class if you so desire, but _now_ I am a bit confused about the complete lack of the entire return object documentation in [zoom api](https://developers.zoom.us/docs/api/rest/reference/phone/methods/#operation/phoneDownloadRecordingTranscript).

As per the calls I have currently made, it would seem that the response consists of:
__string "type"  // probably an enum, but I haven't encountered any values apart from "zoom_transcript"
__int "ver"
__string "recording_id"
__string "meeting_id"
__string "account_id"
__string "host_id"
__datetime "recording_start"
__datetime "recording_end"
__array "timeline" of specific items:
____string "text"
____time "ts" (start time)  // e.g. "00:00:06.760"
____time "end_ts" (end time)
____array of "users"  // might be empty
______string "username"
______boolean "multiple_people"
______string "user_id"
______string "zoom_userid"
______int "client_type"  // most likely an enum as well — but I have to know more for turning it into enum

Does this look convincing?

Reply:
> Does this look convincing?

Like I said, I do not have permission to use any of the "phone" related endpoints so there's no way for me to validate. Therefore, I must rely on you. One thing that seems weird to me is that the response from the Zoom API is not documented:
![image](https://github.com/Jericho/ZoomNet/assets/112710/2bc39068-bc27-4cc9-ae88-46678ff94f5f)

> __string "type" // probably an enum, but I haven't encountered any values apart from "zoom_transcript"

> ______int "client_type" // most likely an enum as well — but I have to know more for turning it into enum

Since Zoom is not documenting the JSON, you probably don't have the information you would need in order to create an enum. That's really unfortunate. You'll have to simply model these properties as string/int. Maybe add a comment so that we know these property should be enums.

Reply:
Yes, like I said, it is not documented. :-)
So you think that it would be best to change JsonObject to the class object with fields like I've observed?

Reply:
Yes indeed, it would be better.

Reply:
Wow, it dumps back to Json _almost_ as it was before — the only difference I see is end timespan now goes after the start timespan (thankfully), and the timespans themselves are "00:00:00.000000" instead of just "00:00:00.000" now. Small pleasures of using libraries.

Reply:
Please check if it is better now.